help/destinations/catalog/crm/outreach.md

```diff
@@ -0,0 +1,208 @@
+
+Outreach settings page for prospects displaying custom fields.
+![Outreach custom fields defaults](../../assets/catalog/crm/outreach/outreach-custom-fields.png)
+
+Outreach settings page for prospects displaying custom fields with *user-friendly* labels matching to the segment names. You can view the segment status on the prospect page against these labels.
+![Outreach custom fields with labels](../../assets/catalog/crm/outreach/outreach-custom-field-labels.png)
+
+> [!NOTE]
+>
+> When updating prospects the label names are not used they are for ease of identification only.
+
+## Supported identities {#supported-identities}
+
+Outreach supports update of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+|Target Identity|Description|Considerations|
+|---|---|---|
+| OutreachId |Custom Outreach identifier that supports mapping of any identity.|Mandatory. You can send any [identity](../../../identity-service/namespaces.md) to the [!DNL Outreach] destination, as long as you map it to the `OutreachId`. |
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | <ul><li>You are exporting all members of a segment, together with the desired schema fields *(for example: email address, phone number, last name)*, according to your field mapping.</li><li> Platform segment statuses are exported to [!DNL Outreach] by specifying their corresponding custom field attribute in [!DNL Outreach], within the **[!UICONTROL Activate Destination]** > **[!UICONTROL Schedule segment export]** > **[!UICONTROL Mapping ID]** field.</li></ul> |
+| Export frequency | **[!UICONTROL Streaming]** | <ul><li>Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations).</li></ul>|
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+> [!IMPORTANT]
+> 
+> To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+![Catalog](../../assets/catalog/crm/outreach/catalog.png)
```

Comment:
This screen has five highlights on it, but users might not understand the order those are shown in. Maybe add some numbering next to them (1. Destinations, 2.Catalog, etc.).